yjit/src/backend/arm64/mod.rs

```diff
@@ -192,6 +192,23 @@ impl Assembler
                         }
                     }
                 },
+                Op::Or => {
+                    match (opnds[0], opnds[1]) {
+                        (Opnd::Reg(_), Opnd::Reg(_)) => {
+                            asm.or(opnds[0], opnds[1]);
+                        },
+                        (reg_opnd @ Opnd::Reg(_), other_opnd) |
+                        (other_opnd, reg_opnd @ Opnd::Reg(_)) => {
+                            let opnd1 = split_bitmask_immediate(asm, other_opnd);
+                            asm.or(reg_opnd, opnd1);
```

Comment:
Thanks for fixing this bug. You should be able to keep the combination of the clause with `Op::And | Op::Or` and use `push_insn` instead of `asm.and`/`asm.or` here.